kore/src/Kore/Attribute/Location.hs

```diff
@@ -0,0 +1,93 @@
+{-|
+Module      : Kore.Attribute.Location
+Description : Line/column location attribute
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+Maintainer  : vladimir.ciobanu@runtimeverification.com
```

Comment:
Could you update docs/attributes.md with both attributes?

Reply:
Done.